src/util/categoryUtils.js

```diff
@@ -135,3 +135,36 @@ export function getHeroChannelCached(client) {
 		return {};
 	}
 }
+
+/**
+ * Get the current selected category name
+ * @param {Number} categoryId
+ * @param {String} name
+ * @return {String} category name
+*/
+export function getCategoryName(categoryId, name) {
```

Comment:
While this file doesn't have utils yet, this would be a good method to add a simple unit test for.